File: bridges/bridge-fabric/v1_21_8/src/main/java/dev/httpmarco/polocloud/bridges/fabric/v1_21_8/mixin/ClientConnectionAccessor.java
package dev.httpmarco.polocloud.bridges.fabric.v1_21_8.mixin;

import net.minecraft.network.ClientConnection;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

import java.net.SocketAddress;

@Mixin(ClientConnection.class)
public interface ClientConnectionAccessor {

    @Accessor("address")
    void setAddress(SocketAddress address);
}

File: bridges/bridge-fabric/src/main/kotlin/dev/httpmarco/polocloud/bridges/fabric/mixin/ClientConnectionAccessor.kt
package dev.httpmarco.polocloud.bridges.fabric.mixin

import net.minecraft.network.ClientConnection
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor
import java.net.InetSocketAddress

@Mixin(ClientConnection::class)
interface ClientConnectionAccessor {

    @set:Accessor("address")
    var address: InetSocketAddress
}